dev/io.openliberty.jakarta.validation.v31_fat/publish/servers/validation.v31.fat/server.xml

```diff
@@ -1,26 +1,26 @@
 <!--
-    Copyright (c) 2023 IBM Corporation and others.
+    Copyright (c) 2023, 2024 IBM Corporation and others.
     All rights reserved. This program and the accompanying materials
     are made available under the terms of the Eclipse Public License 2.0
     which accompanies this distribution, and is available at
     http://www.eclipse.org/legal/epl-2.0/
-    
+
     SPDX-License-Identifier: EPL-2.0
-   
+
     Contributors:
         IBM Corporation - initial API and implementation
  -->
 <server>
 
     <featureManager>
-		<feature>cdi-4.0</feature>
-        <feature>servlet-6.0</feature>
-        <feature>beanValidation-3.0</feature> <!-- TODO rename to validation-3.1 when available-->
+		<feature>cdi-4.1</feature>
+        <feature>servlet-6.1</feature>
+        <feature>beanValidation-3.1</feature> <!-- TODO rename to validation-3.1 when available-->
```

Comment:
Remove the TODO since it is TO-DONE?

Reply:
It is not done.  It might be changed to validation-3.1 like the comment says.  So I left the comment.

Reply:
ok fine